src/Providers/StringMacroServiceProvider.php

```diff
@@ -16,11 +17,17 @@ public function register(): void
 
     public function boot(): void
     {
+        // This macro is depreciated in favor of Str::transformUrlsInvisibly().
+        // As soon as it is not being used anymore, it can be deleted.
```

Comment:
you can already remove it in this PR, it's only used on marketsquare and will be replaced by the new one once this change becomes available

Reply:
done